nni/common/version.py

```diff
@@ -81,3 +81,12 @@ def version_check(expect: dict, raise_error: bool = False) -> None:
                 raise RuntimeError('Version check failed: ' + err_message)
             else:
                 warnings.warn('Version check with warning: ' + err_message)
+
+
+def check_torch_version():
```

Comment:
function name is not clear, maybe `torch_version_is_2`?

Reply:
good suggestion, I will modify it.